devices/b-parasite/b-parasite.json

```diff
@@ -0,0 +1,415 @@
+        "0x0a",
+        "0x0400"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "refresh.interval": 86400,
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 10,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 10,
+            "eval": "Item.val = Attr.val / 2",
+            "fn": "zcl:attr"
+          }
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "config/tholddark",
+          "default": 12000
+        },
+        {
+          "name": "config/tholdoffset",
+          "default": 7000
+        },
+        {
+          "name": "state/dark",
+          "default": false
+        },
+        {
+          "name": "state/daylight",
+          "default": false
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/lightlevel",
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0400",
+            "ep": 10,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/lux",
+          "default": 0
+        }
+      ]
+    }
+  ],
+  "bindings": [
+    {
+      "bind": "unicast",
+      "src.ep": 10,
+      "cl": "0x0402",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x29",
+          "min": 60,
+          "max": 3600,
+          "change": "0x00000032"
+        }
+      ]
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 10,
+      "cl": "0x0405",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x21",
+          "min": 60,
+          "max": 3600,
+          "change": "0x00000032"
+        }
+      ]
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 10,
+      "cl": "0x0408",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x21",
+          "min": 60,
+          "max": 3600,
+          "change": "0x00000032"
+        }
+      ]
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 10,
+      "cl": "0x0400",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x21",
+          "min": 60,
+          "max": 3600,
+          "change": "0x00000005"
```

Comment:
A change of 5 is too tiny in my few. Maybe also take a look at a Philips motion sensor here.

Reply:
Good point (again)! Used the value from the philips motion sensor (2000 ll / ~1.5lx) 36983bb9fc48b6389dd9f3f61be322e6d9571198